doc/api/errors.md

```diff
@@ -1176,6 +1164,14 @@ is set for the `Http2Stream`.
 An option pair is incompatible with each other and can not be used at the same
 time.
 
+<a id="ERR_INPUT_TYPE_NOT_ALLOWED"></a>
+#### ERR_INPUT_TYPE_NOT_ALLOWED
+
+> Stability: 1 - Experimental
+
+The `--input-type` flag was used to attempt to execute a file. This flag can
+only be used with input via `--eval`, `--print` or `STDIN`.
```

Comment:
This shouldn't be an error code, rather it should be an options error thrown in the C++ side.